remote-method/USAGE

```diff
@@ -0,0 +1,6 @@
+Description:
+  Creates a new remote method in the LoopBack application.
+
+Example:
+
+  slc loopback:remote-method
```

Comment:
>@superkhau what are your concerns regarding slc loopback:method?

I don't want to explain inconsistent naming in docs/examples and the often overloaded "function/method" terminlogy in discussions. Unless you deem it is safe to assume that all end users will know `method` = `remote-method`. IMO, I'd rather keep `remote-method` = `Remote method` everywhere. No explaining differences etc.

I'm not strongly opinionated here, so I'm fine with avoiding maintaining aliases. However I do think from a user perspective, it makes sense to have aliases (ie. `npm install -> npm i`), but I think they won't complain too much about having to a type an extra few letters. ;)